dbt_subprojects/dex/models/trades/base/platforms/otsea_base_base_trades.sql

```diff
@@ -0,0 +1,92 @@
+{{ config(
+    schema = 'otsea_base',
+    alias = 'base_trades',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index']
+) }}
+
+WITH token_swaps AS (
+    SELECT
+        evt_block_number AS block_number,
+        CAST(evt_block_time AS timestamp(3) with time zone) AS block_time,
+        evt_tx_from AS maker,
+        evt_tx_to AS taker,
+        swapped AS token_sold_amount_raw,
+        received AS token_bought_amount_raw,
+        CAST(token AS varbinary) AS token_sold_address,
+        -- CAST('0x0000000000000000000000000000000000000000' AS varbinary) AS token_bought_address,
+        0x0000000000000000000000000000000000000000 AS token_bought_address,
+        contract_address AS project_contract_address,
+        evt_tx_hash AS tx_hash,
+        evt_index AS evt_index
+    FROM
+        {{ source('otsea_base', 'OTSeaV2_evt_SwappedTokensForETH') }}
+    {% if is_incremental() %}
+    WHERE
+        {{ incremental_predicate('evt_block_time') }}
+    {% endif %}
+),
+eth_swaps AS (
+    SELECT
+        evt_block_number AS block_number,
+        CAST(evt_block_time AS timestamp(3) with time zone) AS block_time,
+        evt_tx_from AS maker,
+        evt_tx_to AS taker,
+        swapped AS token_sold_amount_raw,
+        received AS token_bought_amount_raw,
+        -- CAST('0x0000000000000000000000000000000000000000' AS varbinary) AS token_sold_address,
+        0x0000000000000000000000000000000000000000 AS token_sold_address,
```

Comment:
Which one is better for `ETH` address on base?
`0x4200000000000000000000000000000000000006`
or
`0x0000000000000000000000000000000000000000`

Reply:
`0x0000000000000000000000000000000000000000` for ETH
`0x4200000000000000000000000000000000000006` for WETH